### ISSUE TITLE
Warning: "bucket_policy_only" deprecated

### ISSUE THREAD

**Issue by amandakarina** [terraform-google-modules/terraform-google-bootstrap#62]
```
Warning: "bucket_policy_only": [DEPRECATED] Please use the uniform_bucket_level_access as this field has been renamed by Google.

  on .terraform/modules/seed_bootstrap/main.tf line 82, in resource "google_storage_bucket" "org_terraform_state":
  82: resource "google_storage_bucket" "org_terraform_state" {

(and one more similar warning elsewhere)
```

**Comment by bharathkkb** [comment]
fixed by #63

**Comment by gojanpaolo** [comment]
Hello! Sorry for reviving this issue but we're having trouble updating google terraform provider from v3 to v4 because when we try to delete `bucket_policy_only` in our terraform state, doing a terraform refresh while still on v3 adds it back. Is there a process to do the update properly? Thanks! Let me know if I need to file a new issue instead
